ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -38,7 +38,7 @@ open class PersonaListView: UITableView {
         case searchDirectory
     }
 
-    enum SearchDirectoryState {
+    public enum SearchDirectoryState {
```

Comment:
>public [](start = 4, length = 6)

same here, @objc and place before private enum

Reply:
I am getting this when adding @objc 
<img width="1375" alt="image" src="https://user-images.githubusercontent.com/67058720/85583344-ce70ce80-b65b-11ea-87ca-a75f6c552d25.png">

Reply:
for enums, instead of adding next to definition, add with a line above:
@objc(MSFSearchDirectoryState). See example: MSFTwoLineTitleViewStyle definition